src/multi_vector_simulator/utils/data_parser.py

```diff
@@ -402,6 +403,9 @@ def convert_epa_params_to_mvs(epa_dict):
                         DATA_TYPE_JSON_KEY
                     ] = TYPE_SERIES
 
+                if asset_group == ENERGY_PRODUCTION:
```

Comment:
Add a TODO to not forget to remove this whenever the EPA tackles it

Reply:
05e0e00b3f09060660d3d5f1853ad071dcaa02a7